ui/apps/dashboard/src/middleware.ts

```diff
@@ -26,9 +26,11 @@ export default authMiddleware({
 
     if (
       isSignedIn &&
+      hasActiveOrganization &&
       !isAccountSetup &&
-      request.nextUrl.pathname !== '/sign-up/account-setup' &&
-      request.nextUrl.pathname !== '/organization-list'
+      request.nextUrl.pathname !== '/organization-list' &&
+      request.nextUrl.pathname !== '/create-organization' &&
+      request.nextUrl.pathname !== '/sign-up/account-setup'
```

Comment:
Worth putting these in a `const` somewhere? I can see this needing to expand is all.

Maybe `const nonRedirectablePaths = ["/organization-list", ...];`.

Reply:
They aren't technically non-redirectable because we redirect to the sign-in page when the user is not signed in. I'm not sure what to call them. I'll try to think of something next when I revisit this middleware after switching to the sync flow.